estela-web/src/components/SpiderDetailPage/index.tsx

```diff
@@ -163,7 +576,7 @@ export class SpiderDetailPage extends Component<RouteComponentProps<RouteParams>
                                     <div className="lg:m-10 md:mx-6 mx-2">
                                         <Row className="flow-root my-6 space-x-4">
                                             <Col className="float-left">
-                                                <Text className="text-[#6C757D] text-xl">My Spider</Text>
+                                                <Text className="text-[#6C757D] text-xl">{name}</Text>
```

Comment:
Don't use custom color [#6C757D] Instead of that, we add tailwind colors, review tailwind.conf file to se estela custom colors

Reply:
Done, changed all colors for the corresponding one in tailwind.config file.